website/docs/reference/directives.md

```diff
@@ -126,6 +126,8 @@ Manually add sources to the project. Does not support chaining, sources are adde
 #### Examples
 `//> using file utils.scala`
 
+`//> using file https://raw.githubusercontent.com/softwaremill/sttp/refs/heads/master/examples/src/main/scala/sttp/client4/examples/json/GetAndParseJsonCatsEffectCirce.scala`
```

Comment:
@Gedochao I think it would be nice to reference a file within `scala-cli` repo itself

Reply:
I mean... dunno if it matters, as long as the example is useful, to be honest. 🤷